Remove Pinst & improve README (#4)
infuy/rif-relay-client: Readme.md

#### How to start

> To start working with this project you need to enable `postinstall` scripts, refer
> to section [Enable postinstall scripts](#enable-postinstall-scripts) to know how to do it. Then just run `npm install` to install
> all the dependencies.

Reviewer: ```suggestion
To start working with this project you need to enable `postinstall` scripts, refer to section [Enable postinstall scripts](#enable-postinstall-scripts) to know how to do it. Then just run `npm install` to install all dependencies.
```

1. Run `npm run dist` to generate the distributable version.
2. Commit and push the dist folder with the updated version to the repository on master.

> **IMPORTANT: when you publish a version postinstall scripts must be disabled. This is disabled by default, don't push
> any change on postinstall scripts section in the `package.json` file.**

Reviewer: ```suggestion
**IMPORTANT: when you publish a version postinstall scripts must be disabled. This is disabled by default, don't push any changes to the postinstall scripts section in the `package.json` file.**
```

#### How to develop

> If you need to modify resources inside this repository the first thing you need to do always is to make sure you have `postinstall` scripts enabled on the `package.json`. These
> are disabled by default due to distribution issues. (This will be solve in the future). This will enable husky and other checks,
> then run `npm install` to execute the post install hooks. After that you can just make your modifications
> and then run `npm run build` to validate them. After you are done with your changes you
> can publish them by creating a distributable version for the consumers.
> 
> #### Enable postinstall scripts
> 
> To enable `postinstall` scripts you need to modify the `package.json` file
> in the section `scripts` and change the line `"_postinstall": "scripts/postinstall",`
> to `"postinstall": "scripts/postinstall",`.

Reviewer: ```suggestion
If you need to modify resources inside this repository the first thing you always need to make sure of is that `postinstall` scripts are enabled in the `package.json` file. These are disabled by default due to distribution issues (which will be solved in the future), but will enable husky and other tools.

Then, run `npm install` to execute the post install hooks. After that, you can just make your modifications and then run `npm run build` to validate them. After you are done with your changes you can publish them by creating a distributable version.

#### Enable postinstall scripts

To enable `postinstall` scripts you need to modify the `package.json` file, specifically the section `scripts` and change the line `"_postinstall": "scripts/postinstall",` to `"postinstall": "scripts/postinstall",`.
```
